pkgs/development/tools/documentation/doxygen/default.nix

```diff
@@ -1,45 +1,24 @@
-{ stdenv, fetchurl, perl, python, flex, bison, qt4, CoreServices, libiconv }:
+{ stdenv, cmake, fetchurl, perl, python, flex, bison, qt4, CoreServices, libiconv }:
 
 let
-  name = "doxygen-1.8.6";
+  name = "doxygen-1.8.11";
 in
 stdenv.mkDerivation {
   inherit name;
 
   src = fetchurl {
     url = "ftp://ftp.stack.nl/pub/users/dimitri/${name}.src.tar.gz";
-    sha256 = "0pskjlkbj76m9ka7zi66yj8ffjcv821izv3qxqyyphf0y0jqcwba";
+    sha256 = "0ja02pm3fpfhc5dkry00kq8mn141cqvdqqpmms373ncbwi38pl35";
   };
 
-  prePatch = ''
-    substituteInPlace configure --replace /usr/bin/install $(type -P install)
-  '';
-
-  patches = [ ./tmake.patch ];
-
   buildInputs =
-    [ perl python flex bison ]
+    [ cmake perl python flex bison ]
```

Comment:
`cmake` is used to build the package and not as a dependency by the package. It therefor belongs to `nativeBuildInputs`.